### DISCUSSION TITLE
Crashing when opening specific maps in editor

### DISCUSSION THREAD

**Discussion by hwabis** [ppy/osu#21138] [Q&A] [answered]
[runtime.log](https://github.com/ppy/osu/files/9946157/runtime.log)

https://osu.ppy.sh/beatmapsets/1327622#taiko/3816540
If I try to open the Kantan, Futsuu, or Muzukashii difficulties (right click from song select --> Edit), then the game crashes. It happens consistently and all other maps open fine (including the other difficulties in the mapset).

**Comment by smoogipoo** [comment, marked as answer]
https://github.com/ppy/osu/issues/21143
